samples/react-company-stories/.yo-rc.json

```
@@ -1,12 +1,14 @@
 {
   "@microsoft/generator-sharepoint": {
+    "plusBeta": false,
     "isCreatingSolution": true,
     "environment": "spo",
     "version": "1.11.0",
```

Don't forget to update this property to 1.14.0
```suggestion
    "version": "1.14.0",
```